SeleniumDemoQA/SeleniumDemoQA/Pages/FormPage.cs

```diff
@@ -33,41 +32,92 @@ public FormPage(IWebDriver driver) : base(driver)
 
         public void FillFirstName(string firstName)
         {
-            FillInput(firstNameInputBy, firstName);
+            _driver.FillInput(firstNameInputBy, firstName);
         }
 
         public void FillLastName(string lastName)
         {
-            FillInput(lastNameInputBy, lastName);
+            _driver.FillInput(lastNameInputBy, lastName);
         }
 
         public void FillEmail(string email)
         {
-            FillInput(emailInputBy, email);
+            _driver.FillInput(emailInputBy, email);
+        }
+
+        public void ClickGenderRadio()
+        {
+            _driver.ClickElement(genderRadioBy);
         }
 
         public void FillMobileNumber(string mobileNumber)
         {
-            FillInput(mobileNumberInputBy, mobileNumber);
+            _driver.FillInput(mobileNumberInputBy, mobileNumber);
+        }
+
+        public void ClickDateOfBirth()
```

Comment:
It is better to create function SetDateOfBirth(string month, string year, string day) which will include actions from ClickDateOfBirth, SelectMonth, SelectYear, SelectDate